api/cf-rpc-apis/Cargo.toml

```diff
@@ -18,5 +18,6 @@ jsonrpsee = { workspace = true, features = ["full"] }
 cf-chains = { workspace = true, default-features = true }
 cf-primitives = { workspace = true, default-features = true }
 cf-rpc-types = { workspace = true }
+state-chain-runtime = { workspace = true }
```

Comment:
I think one of the goals of the rpc-apis crate is to avoid dependencies on the runtime... can we avoid it?

cc @nmammeri